cards.c

```diff
@@ -0,0 +1,77 @@
+/* Jared Wasserman -- cards.c */
+/*This program random chooses 5 cards and prints them out. It uses structs and enums.*/
+
+#include <stdio.h>
```

Comment:
Perfect job! The only suggestion I have is to include comments above each of your functions!